### PR TITLE
add ability to __construct Picqer\Financials\Exact\Model with attributes

### PR DESCRIPTION
**Pull request by kirkita**

### CODE REVIEW

File: src/LaravelExactOnline.php [hunk 1]
```diff
@@ -458,7 +458,7 @@ public static function loadConfig()
             );
         }
 
-        return (object) json_decode($config, false);
+        return (object)json_decode($config, false);
```
Comment (websmurf) [inline src/LaravelExactOnline.php]: Could you please revert this change to keep the current code style?

File: src/LaravelExactOnline.php [hunk 2]
```diff
@@ -362,7 +362,7 @@ public function __call($method, $arguments)
             throw new RuntimeException("Invalid type called");
         }
 
-        return new $classname($this->connection());
+        return empty($arguments) ? new $classname($this->connection()) : new $classname($this->connection(), $arguments[0]);
```
Comment (websmurf) [inline src/LaravelExactOnline.php]: What would be the use case of only passing along the first argument?

File: src/LaravelExactOnline.php (hunk 1 above)
Comment (kirkita) [reply src/LaravelExactOnline.php]: sure <3

File: src/LaravelExactOnline.php (hunk 2 above)
Comment (kirkita) [reply src/LaravelExactOnline.php]: Picqer\Financials\Exact\Model's __construct method accepts second parameters as array which should be one dimensional array with attributes and values. 
For example if you call  $exactOnline->Account(['ID'=>'0000-....-0000']) the `$argument` will be two dimensional array and passing directly will not have any effect.
